CHANGES.rst

```diff
@@ -230,6 +230,9 @@ Bug fixes
 
 - ``astropy.modeling``
 
+  - Fix a bug to allow instantiation of a modeling class having a parameter
+    with a custom setter that takes two parameters ``(value, model)`` [#4586]
+
```

Comment:
Please move this out of the v1.2 section and instead copy it to the v1.0.9 section or whatever the latest v1.1.x section is, since it's a bug fix.